packages/fuels-contract/src/script.rs

```diff
@@ -46,6 +47,8 @@ impl Script {
         Self { tx }
     }
 
+    //builder pattern -> data & opcode, set_inputs and outputs, call after build.
+
```

Comment:
Reformulate, maybe use a sentence with an example as we do in other places

Reply:
This was plan i forgot to delete!